src/styles/stars.scss

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(../images/star.svg);
+}
+
+.stars_star,
+.stars--4 .stars_star:nth-child(-n+4) {
```

is it necessary here?

```suggestion
{
```